adafruit_sharpmemorydisplay.py

```diff
@@ -27,13 +27,16 @@
 
 """
 # pylint: enable=line-too-long
+from __future__ import annotations
 
 import adafruit_framebuf
 from adafruit_bus_device.spi_device import SPIDevice
 from micropython import const
 
 try:
+    from PIL import Image
```

Comment:
We have a stubbed out version of Image that can be used for typing so that the library won't depend on the full PIL module. https://github.com/adafruit/Adafruit_CircuitPython_Typing/blob/e54126d5e9a504775a5d4369beb8b2aae1fe8963/circuitpython_typing/pil.py#L26

It can be imported like this:
```
from circuitpython_typing.pil import Image
```
Then used in the typing annotation the same way as the PIL one.